packages/osd-test/src/failed_tests_reporter/run_failed_tests_reporter_cli.ts

```diff
@@ -65,9 +65,9 @@ export function runFailedTestsReporterCli() {
         }
 
         const isPr = !!process.env.ghprbPullId;
-        const isMasterOrVersion = branch === 'master' || branch.match(/^\d+\.(x|\d+)$/);
+        const isMasterOrVersion = branch === 'main' || branch.match(/^\d+\.(x|\d+)$/);
```

Comment:
is this the only place this constant is used? If so we can probably change the constant as well to `isMainOrVersion`